ivy/functional/ivy/data_type.py

```diff
@@ -917,21 +917,38 @@ def default_float_dtype(
     float_dtype: Optional[Union[ivy.FloatDtype, ivy.NativeDtype]] = None,
     as_native: Optional[bool] = None,
 ) -> Union[ivy.Dtype, str, ivy.NativeDtype]:
-    """Summary.
-
+    """
     Parameters
     ----------
     input
-         (Default value = None)
+       (Default value = None) Number or array for inferring default float
+       dtype.
     float_dtype
-
+       (Default value = None) float type to be returned as default.
     as_native
-         (Default value = None)
+         (Default value = None) Whether to return the default float dtype as
+         native dtype.
 
     Returns
     -------
-        Return the input float dtype if provided, otherwise return the global default
-        float dtype.
+        Return the input float dtype if provided, otherwise return the global
```

Comment:
I meant the actual docstring, not the docstring examples😅. Right now "Returns" doesn't fully describe what the function does and it could cause confusion. But it's great how you updated the examples as well!

Reply:
Oh ok, I modfied it. Is the docstring of the return thorough enough now?

Reply:
Yes, that's great 🙂